_tests_/itinerary.test.js

```diff
@@ -0,0 +1,14 @@
+const Itinerary = require('../src/itinerary.js');
```

Comment:
Not to do with this file, but there really should be a readme included in the PR, even a basic outline of what this does and how to install it would have been good as a starter (if pushed for time putting a few 'todo's would help give the reviewer / user something to go off)